### PR TITLE
Updating svg-to-raster when no resolutions are passed in as the iterants

### PR DESCRIPTION
**Pull request by ramitha**
- In this case, we assume that the resolution needs to be itself, i.e., 1
- We also rename the icon so that the resolution is present at the end

### CODE REVIEW

File: packages/@icon-magic/generate/src/plugins/svg-to-raster.ts [hunk 1]
```diff
@@ -104,14 +114,36 @@ export const svgToRaster: GeneratePlugin = {
...
+      // resolution is at the end of the flavor's name Eg: if flavor.name =
+      // home-filled@12 then the resulting name will be
+      // home-filled-width-height@12 instead of home-filled@12-width-height
+      let res;
+      let assetName;
+      if (!params.propCombo.resolutions) {
+        res = 1;
+        const resolutionFromName = flavor.name.match(/@[0-9|\.]*/);
+        if (resolutionFromName) {
+          // strip the resolution from the falvor's name and append it to the end
```
Comment (oa495) [inline packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: nit: flavor

File: packages/@icon-magic/generate/src/plugins/svg-to-raster.ts [hunk 2]
```diff
@@ -104,14 +114,36 @@ export const svgToRaster: GeneratePlugin = {
...
+          assetName = `${flavor.name.replace(
+            resolutionFromName[0],
+            ''
+          )}-${w}x${h}${resolutionFromName[0]}`;
+        } else {
+          // appends @1 to the name
+          assetName = `${flavor.name}-${w}x${h}@${res}`;
+        }
+      } else {
+        res = params.propCombo.resolutions;
```
Comment (oa495) [inline packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: is this not an array?

Comment (ramitha) [reply packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: Not when it reaches the plugin. It'll be an array in the plugin-manager, but there will only be one combination of properties that is sent down to the plugin at a time

File: packages/@icon-magic/generate/src/plugins/svg-to-raster.ts (hunk 1 above)
Comment (ramitha) [reply packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: Fixed

**Pull request merged**